code/game/objects/items/robot/items/tools.dm

```diff
@@ -174,5 +176,199 @@
 	projectile.speed *= (1 / projectile_speed_coefficient)
 	projectile.cut_overlay(projectile_effect)
 
+//bare minimum omni-toolset for modularity
+/obj/item/borg/cyborg_omnitool
+	name = "cyborg omni-toolset"
+	desc = "You shouldn't see this in-game normally."
+	icon = 'icons/mob/silicon/robot_items.dmi'
+	icon_state = "toolkit_medborg"
+	///our tools
+	var/list/radial_menu_options = list()
+	///The toolspeed for our tools
+	toolspeed = 1
+	var/obj/item/reference
```

Comment:
Tool speed shouldnt get a comment (should be commented on parent)
reference should